meetings/minutes/weekly-2020-12-04.md

```diff
@@ -0,0 +1,35 @@
+# Weekly meeting
+
+**2020-12-04**
+
+Chair: @YangKeao
+
+## New features
+
```

Comment:
```suggestion
* Async Commit
  * Redesign fallback from async commit / 1PC pingcap/tidb#21091, tikv/tikv#9181
    * Improve a UTF to cover fallback: automated-tests#535
  * A few unit tests tikv/tikv#9183
  * Use system variable instead of config file to enable async commit / 1PC pingcap/tidb#21365, pingcap/tidb#21438
    * Done: UTF, TiPocket changes
    * Todo: Jepsen
  * New issues
    * Make single region 1PC compatible with CDC and other components
    * Compatibility between async commit fallback and CDC
    * Compatibility issue between async commit / 1PC and GC in Compaction Filter

```